### PR TITLE
include nix install script in the website and remove the redirect (fo…

### PR DESCRIPTION
**Pull request by garbas**
…r now)

also:
- use GITHUB_TOKEN instead of custom GH_TOKEN
- don't create comment on cron jobs

### CODE REVIEW

**Comment by samueldr** [comment]
For those out of the loop, and history spelunkers, can the reason for keeping the script as a website artifact, rather than a redirect, be added to either a comment to the Netlify config, or described in the PR body?

I'm saying this since I don't know, and I've been kinda following what's going on.

File: .github/workflows/cron.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: "Hourly Build & Deploy to Netlify"
```
Comment (edolstra) [inline .github/workflows/cron.yml]: What the difference between this and `.github/workflows/main.yml` (which also has a `cron: '0 * * * *'` line)?

**Comment by edolstra** [comment]
We probably also need to serve `https://nixos.org/releases/nix/nix-<version>/install` directly rather than as a redirect, since there are people who are installing it without `-L`. (We won't need to do that for new releases, just the existing ones.)

**Comment by edolstra** [comment]
@samueldr The reason is that a lot of people are doing `curl https://nixos.org/nix/install` rather than `curl -L ...`, so they break if the file becomes a redirect.

**Comment by edolstra** [comment]
BTW breaking `curl https://nixos.org/nix/install` is not really a big deal since people shouldn't be doing that anyway in scripts (because the version could change in incompatible ways at any time). But the version-specific scripts (`https://nixos.org/releases/nix/nix-<version>/install`) should continue to work.

File: .github/workflows/cron.yml (hunk 1 above)
Comment (garbas) [reply .github/workflows/cron.yml]: oh sorry, i was suppose to remove cron from `main.yml`. i'll fix it.

**Comment by garbas** [comment]
@edolstra i will create the redirects. but going forward people should rely on https://releases.nixos.org/nix/nix-<version>/install. I will include this into announcement.

File: .github/workflows/cron.yml (hunk 1 above)
Comment (edolstra) [reply .github/workflows/cron.yml]: I mean what's the difference? They look very similar. Do we need both?

**Comment by garbas** [comment]
oh actually ... above should also just follow the redirect.

File: .github/workflows/cron.yml (hunk 1 above)
Comment (garbas) [reply .github/workflows/cron.yml]: in cron.yml I also remove `github-token` to not add comments every hour. it makes listening to notifications on this github repo almost useless. this is the main reason for the split.

**Pull request merged**